src/mmw/apps/bigcz/clients/cuahsi.py

```diff
@@ -45,16 +77,43 @@ def find_service(services, service_code):
     return None
 
 
-def parse_records(sites, services):
+def parse_records(series, services):
+    """
+    Join series catalog records to corresponding service.
+    """
     result = []
-    for site in sites:
-        service = find_service(services, site['servCode'])
+    for record in series:
+        service = find_service(services, record['ServCode'])
         if service:
-            record = parse_record(site, service)
+            record = parse_record(record, service)
             result.append(record)
     return result
 
 
+def group_series_by_location(series):
```

Comment:
I'm open to finding a better term for this. `GetSeriesCatalogForBox2` returns multiple variables per site/location, but we don't want to display individual variables-- just sites. So this groups each series by location and returns fields common to all variables. There are no aggregated values so we just store the first record and discard subsequent records.

This was implemented based on my interpretation of [this comment](https://github.com/WikiWatershed/model-my-watershed/issues/1931#issuecomment-306838234).